src/sketchybar.c

```diff
@@ -372,6 +385,40 @@ void generate_uid(char* buffer) {
   snprintf(buffer, 64, "item_%d", g_uid_counter++);
 }
 
+int push(lua_State *state) {
+  // Ensure push is in a valid state:
+  if (lua_gettop(state) < 2) {
+    char error[] = "[Lua] Error: expecting at least two arguments for 'push'";
+    printf("%s. Received %d\n", error, lua_gettop(state));
+    return 0;
+  }
+  if (lua_type(state, 1) != LUA_TSTRING && lua_type(state, 1) != LUA_TTABLE) {
+    char error[] = "[Lua] Error: expecting a 'string' or a 'table' as the first "
+                   "argument for 'push'";
+    printf("%s. Found '%s'\n", error, luat_to_string(lua_type(state, 1)));
+    return 0;
+  } else if (lua_type(state, 2) != LUA_TTABLE) {
+    char error[] = "[Lua] Error: expecting a 'table' as the second argument for 'push'";
+    printf("%s. Found '%s'\n", error, luat_to_string(lua_type(state, 2)));
+  }
+
+  // Technically this method will work regardless, so all we need to do 
+  // is ensure a valid input state before we reach this part:
+  struct stack *stack = stack_create();
+  stack_init(stack);
+
+  const char *name = get_name_from_state(state);
+  parse_table_values_to_stack(state, 2, stack);
+
+  stack_push(stack, name);
+  stack_push(stack, PUSH);
+  char *response = sketchybar(stack);
+  stack_destroy(stack);
+  if (response) free(response);
+
+  return 1;
```

Comment:
We are not pushing any response, so I think we should return 0 here.

Reply:
Resolved in "a8fb69c"